README.md

```diff
@@ -8,42 +14,59 @@
 <img src="https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot7%202016-09-02.png" width="50%">
 <img src="https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot6%202016-04-17.png" width="50%">
 
+## Table of Contents
+
+- [Project Status](#project-status)
+- [Run](#run)
+  - [Live Demo](#live-demo)
+  - [Desktop](#desktop)
+  - [Browser](#browser)
+- [Development](#development)
+  - [Requirements](#requirements)
+  - [Get the source code](#get-the-source-code)
+  - [Install dependencies](#install-dependencies)
+  - [Run Tests](#run-tests)
+  - [Browser app development](#browser-app-development)
+    - [Build](#build)
+  - [Desktop app development](#desktop-app-development)
+    - [Build](#build-1)
+- [Contributing](#contributing)
+- [License](#license)
+
 ## Project Status
 
 **Status:** *In active development*
 
 Check the project's [roadmap](https://github.com/haadcode/orbit/blob/master/ROADMAP.md) to see what's happening at the moment and what's planned next.
 
-[![Project Status](https://badge.waffle.io/haadcode/orbit.svg?label=In%20Progress&title=In%20Progress)](http://waffle.io/haadcode/orbit)
-[![CircleCI Status](https://circleci.com/gh/haadcode/orbit.svg?style=shield&circle-token=158cdbe02f9dc4ca4cf84d8f54a8b17b4ed881a1)](https://circleci.com/gh/haadcode/orbit)
-
-See also [CHANGELOG](https://github.com/haadcode/orbit/blob/master/CHANGELOG.md) for what's new!
+See also the [CHANGELOG](https://github.com/haadcode/orbit/blob/master/CHANGELOG.md) for what's new!
 
 ## Run
 
 Orbit can be run either in a browser or as a native desktop application. The browser application uses [js-ipfs](http://github.com/ipfs/js-ipfs) implementation of [IPFS](http://ipfs.io). The desktop version uses the [go-ipfs](https://github.com/ipfs/go-ipfs) implementation of [IPFS](http://ipfs.io) together with [Electron](http://electron.atom.io/).
 
 ***Please note that Orbit is not secure at the moment!***
 
-#### Live Demo
+### Live Demo
 
 [http://orbit.libp2p.io/](http://orbit.libp2p.io)
 
 *The live demo is an old version of Orbit. It is recommended to follow the instructions below to run the latest version.*
 
-#### Desktop
-```
+### Desktop
+
+```sh
```

Comment:
I've spent a while going through arguments for and against syntax highlighting online, and it looks like this is 100% down to preference. So, I'm happy to remove them; but we're using them in the rest of IPFS, and my strongest argument for keeping them may be that not spending time on divergent standards is better for the organization as a whole. As I think you're not going with standard-readme for your readmes, you should write your opinion up elsewhere, but it may result in more bikeshed-color conversations in the future.